addons/sourcemod/configs/szf/weapons.cfg

```diff
@@ -1,5 +1,365 @@
 "Weapons"
 {
+	"general"
+	{
+		// "model"		REQUIRED - weapon's model that appears in the world
+		// "rarity"		Weapon's rarity ("common", "uncommon", "rare", "pickup")
+		// "name"		Weapon's name (appears in chat if the weapon is rare)
```

Comment:
Reason i added the `name` is because of `TF2Econ_GetItemName` not giving correct name for stock weapon and i've never really find myself a proper fix for it.

Though DFS have a fix for it and open source, shouldn't be hard to use his stuffs for this (it even translates whatever translation your using)